docs/design.md

```diff
@@ -112,7 +112,7 @@ Spec:
       target: xyz
       type: bridge
     - source:
-        name: ovirtmgmt
+        name: ovirtmgmt/ovirtmgmt
```

Comment:
What do you think about making the names reflect that they correspond to vnic profile and network names (i.e -net, -vnic-profile suffixes or something similar)?

Reply:
specifically for `ovirtmgmt/ovirtmgmt` I'd like to have it as is, since they are default values in ovirt and changing it to ovirtmgmt-net might be misleading. I'll add a description for that value.
The mapping above it relies to what you suggested `red/profile1`